client/packages/common/src/ui/components/inputs/Filters/FilterMenu.tsx

```diff
@@ -42,7 +42,7 @@ interface FilterDefinitions {
 
 // CONSTANTS
 const RESET_KEYWORD = 'RESET';
-export const FILTER_WIDTH = 220;
+export const FILTER_WIDTH = 180;
```

Comment:
I couldn't find any filters that really needed this additional width, although we should keep a lookout as this applies everywhere 😬